vision/classification_and_detection/python/backend_onnxruntime.py

```diff
@@ -28,14 +28,13 @@ def load(self, model_path, inputs=None, outputs=None):
         """Load model and find input/outputs from the model file."""
         opt = rt.SessionOptions()
         # enable level 3 optimizations
-        # FIXME: enable below once onnxruntime 0.5 is released
-        # opt.set_graph_optimization_level(3)
+        # Disable for onnxruntime < 0.5
```

Comment:
Is it time to enable this?

Reply:
I tried it but unfortunately it resulted in slow down of >10%.